frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -64,6 +92,23 @@ export default {
       type: Array,
       default: () => [],
     },
+    header: {
+      type: String,
+      required: false,
+      default: undefined
+    },
+    showInfoMessage: {
+      type: Boolean,
+      default: true,
+    },
+    showTableHeader: {
+      type: Boolean,
+      default: true,
+    },
+    showRequiredMessage: {
```

Comment:
This may not be required given my comments on AppMissingInfoError.vue.

Reply:
This is not related to AppMIssingInfoError.vue... the property is used to flag display of a required message.

Reply:
removed due to better design of using slot for AppMissingInfoError.